net/query.go

```diff
@@ -3,14 +3,25 @@ package net
 import (
 	"net/http"
 	"net/url"
+
+	log "github.com/sirupsen/logrus"
 )
 
 type ValidateQueryHandler struct {
-	Handler http.Handler
+	LogsEnabled bool
+	LogLevel    string
+	Handler     http.Handler
 }
 
 func (q *ValidateQueryHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if _, err := url.ParseQuery(r.URL.RawQuery); err != nil {
+
+		if q.LogsEnabled {
+			logLevel, _ := log.ParseLevel(q.LogLevel)
+			log.SetLevel(logLevel)
```

Comment:
This chnages global logger level which is wrong.
It should be something like
```go
switch q.level {
case "DEBUG":
  log.Debugf(...)
case "INFO":
  log.Infof(...)
...
}
```
or even
```go
log.StandardLogger().Logf(q.level, "invalid query: %v", err)
```